### PR TITLE
[11.0] [MIG] web_favicon

### PR DESCRIPTION
**Pull request by misern2**
Standard Migration

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together "OCA Transbot..." commits.

**Comment by misern2** [comment]
@pedrobaeza  done!

**Review by yajo — CHANGES_REQUESTED** [review]
Travis is failing: https://travis-ci.org/OCA/web/jobs/302012013#L901

**Comment by yajo** [comment]
Can't we just apply website's favicon into backend? Odoo 10+ already has this setting built in:

captura de pantalla de 2017-11-16 09-17-37

BTW you have failures, check travis.

**Comment by pedrobaeza** [comment]
I prefer to have a separate one.

**Comment by misern2** [comment]
@pedrobaeza @Yajo Travis fixed.

**Review by yajo — APPROVED** [review]

File: web_favicon/controllers/web_favicon.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from odoo import http
+from odoo.tools.misc import file_open
+
+
+class WebFavicon(http.Controller):
+
+    @http.route('/web_favicon/favicon', type='http', auth="none")
+    def icon(self):
+        request = http.request
+        if 'uid' in request.env.context:
```
Comment (simahawk) [inline web_favicon/controllers/web_favicon.py:17]: wouldn't be better to check the session? 
```python
if request.session.uid:
    company = request.env.user.company_id
```

**Review by simahawk — APPROVED** [review]

File: web_favicon/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (phatnguyenuit) [inline web_favicon/__init__.py:1]: In python3 `# -*- coding: utf-8 -*-` it should be removed.

Same for the others python file in this pull request.

**Review by phatnguyenuit — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
As there are no blocking comments, I merge.

**Pull request merged**